test/draw/RasterBgrTest.ooc

```diff
@@ -45,7 +45,27 @@ RasterBgrTest: class extends Fixture {
 			expect(image2 distance(image3), is equal to(0.0f))
 			image1 free(); image2 free(); image3 free()
 		})
+		this add("swapped RB", func {
+			output := "test/draw/output/rbswapped.png"
+			image := RasterBgr open(this sourceFlower)
+			image2 := RasterBgr open(this sourceFlower)
+			image swapRedBlue()
+			image save(output)
+			for (row in 0 .. image height)
+				for (column in 0 .. image width) {
+					pixel1 := image[column, row]
+					pixel2 := image2[column, row]
+					expect(pixel1 red == pixel2 blue)
+					expect(pixel1 green == pixel2 green)
+					expect(pixel1 blue == pixel2 red)
+				}
+			image free()
+			image2 free()
+			output free()
+		})
 	}
 }
 
-RasterBgrTest new() run()
\ No newline at end of file
+test := RasterBgrTest new()
+test run()
+test free()
```

Comment:
The last one. No variables for something we're not keeping anyway, fits on one line.